### ISSUE TITLE
Only historic=boundary_stone, no boundary=marker

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#8509]
https://wiki.openstreetmap.org/wiki/Tag:boundary%3Dmarker could even be a remote corner of a ranch. A very important point to map. However not a historic=boundary_stone. Therefore iD needs to be more flexible.

**Comment by matkoniecz** [comment]
this issue should be moved to https://github.com/openstreetmap/id-tagging-schema

**Comment by jidanni** [comment]
https://ux.stackexchange.com/questions/155316/how-to-improve-flawed-blocking-algorithm caused me not to know you commented.

I don't know how to move GitHub issues.

Should I created a new issue there?

Please turn off blocking, so I know when you reply.

**Comment by bryceco** [comment]
You should create a new issue there, add a comment here with a link saying you moved it, and then close this issue.

**Comment by matkoniecz** [comment]
> Please turn off blocking, so I know when you reply.

no, as I do not want to get issues from you on my projects due to their low quality and high quantity
